<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddFrinedorGroup</class>
 <widget class="QWidget" name="AddFrinedorGroup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>451</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">
QLineEdit {
    border-width: 1px;
    border-style: solid;
	border-color:rgb(199, 199, 199);
	background-color:rgb(255, 255, 255);
}
QComboBox {
    border-width: 1px;
    border-style: solid;
    border-radius: 4px;
	border-color:rgb(199, 199, 199);
	background-color:rgb(255, 255, 255);
}
QFrame{
};
background-color:rgb(255, 255, 255);
border:1px solid rgba(227, 135, 193, 255);
background-color: rgb(255, 255, 255);
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="frame_2">
       <property name="geometry">
        <rect>
         <x>-1</x>
         <y>-1</y>
         <width>451</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(227, 135, 193, 255), stop:1 rgba(251, 171, 182, 255));
border: 1px solid rgba(255 , 255 , 255, 0);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
      <widget class="QFrame" name="frame_3">
       <property name="geometry">
        <rect>
         <x>49</x>
         <y>59</y>
         <width>351</width>
         <height>151</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;
background:none;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>50</y>
          <width>211</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit
{
    background:white;
    padding-left:5px ;
    padding-top:1px ;
    border-bottom-left-radius:3px;
    border-bottom-right-radius:3px;
    border: 1px solid rgb(209 , 209 , 209);
}
QLineEdit:hover
{
    padding-top:0px ;
    border: 1px solid rgb(255, 192, 207);
}</string>
        </property>
       </widget>
       <widget class="QPushButton" name="AddFriendButton">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>110</y>
          <width>112</width>
          <height>32</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(227, 135, 193, 255), stop:1 rgba(251, 171, 182, 255));
color:rgb(255,255,255);
border-radius: 5px;
border:none;
}QPushButton::hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:1 rgba(227, 135, 193, 255), stop:0 rgba(251, 171, 182, 255));
}
QPushButton::pressed{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(227, 135, 193, 255), stop:1 rgba(251, 171, 182, 255));
}</string>
        </property>
        <property name="text">
         <string>查找用户</string>
        </property>
       </widget>
       <widget class="QPushButton" name="AddGroupButton">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>110</y>
          <width>112</width>
          <height>32</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(227, 135, 193, 255), stop:1 rgba(251, 171, 182, 255));
color:rgb(255,255,255);
border-radius: 5px;
border:none;
}QPushButton::hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:1 rgba(227, 135, 193, 255), stop:0 rgba(251, 171, 182, 255));
}
QPushButton::pressed{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(227, 135, 193, 255), stop:1 rgba(251, 171, 182, 255));
}</string>
        </property>
        <property name="text">
         <string>查找群聊</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="textEdit">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>10</y>
          <width>271</width>
          <height>31</height>
         </rect>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'.AppleSystemUIFont'; font-size:14pt;&quot;&gt;请输入用户/群聊的Jid&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
